staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/validation.go

```diff
@@ -17,12 +17,27 @@ limitations under the License.
 package validation
 
 import (
+	"github.com/go-openapi/strfmt"
 	"github.com/go-openapi/spec"
 	"github.com/go-openapi/validate"
 
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
 )
 
+// NewValidator creates an openapi schema validator for the given CRD validation.
+func NewValidator(schema *apiextensions.JSONSchemaProps) (*validate.SchemaValidator, error) {
+	// Convert CRD schema to openapi schema
+	openapiSchema := &spec.Schema{}
+	if err := convertJSONSchemaProps(schema, openapiSchema); err != nil {
```

Comment:
Can you remove `ConvertToOpenAPITypes` since that's not really required anymore?

Reply:
removed